browser/app/js/components/Browse.js

```diff
@@ -480,338 +486,309 @@ export default class Browse extends React.Component {
+              <input type="file" onChange={ this.uploadFile.bind (this) } id="object-upload-input" />
+              <label htmlFor="object-upload-input"> </label>
+            </a>
+          </Dropdown.Menu>
+        </Dropdown>
+    }
+
+    let deleteButton =''
+    if (web.LoggedIn ()) {
+      deleteButton = <button onClick={ this.shareObject.bind (this) } disabled={ checkedObjects.length != 1 } className="zmdi zmdi-share" />
     }
 
     return (
-      <div className={ classNames({
-                   'file-explorer': true,
-                   'toggled': sidebarStatus
-                 }) }>
-        <SideBar searchBuckets={ this.searchBuckets.bind(this) }
-          selectBucket={ this.selectBucket.bind(this) }
-          clickOutside={ this.hideSidebar.bind(this) }
-          showPolicy={ this.showBucketPolicy.bind(this) } />
-        <div className="fe-body">
-          <div className={ 'list-actions' + (classNames({
-                             ' list-actions-toggled': checkedObjects.length > 0
-                           })) }>
-            <span className="la-label"><i className="fa fa-check-circle" /> { checkedObjects.length } Objects selected</span>
-            <span className="la-actions pull-right"><button onClick={ this.downloadSelected.bind(this) }> Download all as zip </button></span>
-            <span className="la-actions pull-right"><button onClick={ this.showDeleteConfirmation.bind(this) }> Delete selected </button></span>
-            <i className="la-close fa fa-times" onClick={ this.clearSelected.bind(this) }></i>
-          </div>
-          <Dropzone>
-            { alertBox }
-            <header className="fe-header-mobile hidden-lg hidden-md">
-              <div id="feh-trigger" className={ 'feh-trigger ' + (classNames({
-                                                  'feht-toggled': sidebarStatus
-                                                })) } onClick={ this.toggleSidebar.bind(this, !sidebarStatus) }>
-                <div className="feht-lines">
-                  <div className="top"></div>
-                  <div className="center"></div>
-                  <div className="bottom"></div>
-                </div>
+      <section className="browser__inner">
+        { alertBox }
+
+        <section className={ classNames({ 'content': true, 'content--toggled': sidebarStatus }) }>
+
+          <header className="header">
+            <div className="toolbar">
+              <div className="actions">
+                <button className="zmdi zmdi-menu" onClick={ this.toggleSidebar.bind(this, !sidebarStatus) }/>
+                <button className="zmdi zmdi-view-comfy" />
+
+                <button onClick={ this.showDeleteConfirmation.bind (this) } disabled={ checkedObjects.length == 0 } className="zmdi zmdi-delete" />
+                { deleteButton }
+                <button onClick={ this.downloadSelected.bind (this) } disabled={ checkedObjects.length == 0 } className="zmdi zmdi-download" />
```

Comment:
This button doesn't do anything when pushed with objects selected on my mac, is it supposed to work?